.github/workflows/install_isis_test.yml

```diff
@@ -0,0 +1,73 @@
+name: Test install_isis.sh
+
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - 'isis/scripts/install_isis.sh'
+      - '.github/workflows/install_isis_test.yml'
+  pull_request:
+    branches:
+      - dev
+    paths:
+      - 'isis/scripts/install_isis.sh'
+      - '.github/workflows/install_isis_test.yml'
+  workflow_dispatch:
+
+jobs:
+  test-install-isis:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-14]
+        test_case:
+          - lts
+          - dev
+          - main
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Make install_isis.sh executable
+        run: chmod +x ./isis/scripts/install_isis.sh
+
+      - name: Run install_isis.sh with test case
+        run: |
+          mkdir -p /tmp/isis_test_prefix
+          set -e
+          case "${{ matrix.test_case }}" in
+            lts)
+              echo "Running install_isis.sh with LTS environment"
+              bash ./isis/scripts/install_isis.sh -l lts -v latest -n auto --no-data --install-prefix /tmp/isis_test_prefix
+              ;;
+            dev)
+              echo "Running install_isis.sh with dev environment"
+              EXPECTED_ENV_NAME="isis-dev"
+              bash ./isis/scripts/install_isis.sh -l dev -v latest --no-data -n $EXPECTED_ENV_NAME --install-prefix /tmp/isis_test_prefix
+              ;;
+            main)
+              echo "Running install_isis.sh with main environment"
+              bash ./isis/scripts/install_isis.sh -l main -v latest -n auto --install-prefix /tmp/isis_test_prefix
+              ;;
+            *)
+              echo "Unknown test case"
+              exit 1
+              ;;
+          esac
+
+
+      - name: Check ISIS installation
+        run: |
+          echo "Checking ISIS installation at /tmp/isis_test_prefix/"
+          ls /tmp/isis_test_prefix/
+          source ~/.bashrc
+          ISIS_ENV_PATH=$(find /tmp/isis_test_prefix -mindepth 1 -maxdepth 1 -type d | head -n 1)
+          # weird linux thing
+          cp "$ISIS_ENV_PATH/lib/libmkl_rt.so" "$ISIS_ENV_PATH/lib/libmkl_rt.so.2" || true
```

Comment:
This is likely going to bite us later. Shouldn't we consider the mkl issue to be a valid problem with the current release rather than fixing it like this?